docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md

```diff
@@ -20,10 +20,11 @@ To start feature integration, integrate the required features:
 | Spryker Core | {{page.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/spryker-core-feature-integration.html) |
 | Order Management | {{page.version}} | [Order Management feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/order-management-feature-integration.html) |
 | State Machine | {{page.version}} | [State Machine feature integration](https://github.com/spryker-feature/state-machine) |
+| Marketplace Dummy Payment | {{page.version}} | [Marketplace Dummy Payment feature integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-dummy-payment-feature-integration.html) |
```

Comment:
```suggestion
| Marketplace Dummy Payment | {{page.version}} | [Marketplace Dummy Payment integration](/docs/marketplace/dev/feature-integration-guides/{{page.version}}/marketplace-dummy-payment-feature-integration.html) |
```

Reply:
Isn't "spryker/dummy-marketplace-payment" a feature?

Reply:
Nope, it is just a module, see https://release.spryker.com/features for the complete feature list.